src/app/api/feed/route.ts

```
@@ -0,0 +1,184 @@
+    return NextResponse.json({ message: "Unauthorized" }, { status: 401 });
+  }
+
+  try {
+    const { searchParams } = new URL(request.url);
+    const cursor = searchParams.get("cursor");
+    const limit = Math.min(
+      parseInt(searchParams.get("limit") || "20", 10),
+      50 // Maximum limit to prevent abuse
+    );
+
+    if (isNaN(limit) || limit < 1) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)